app/controllers/events_controller.rb

```diff
@@ -9,6 +9,8 @@ def show
 
   def new
     @event = Event.new
+    @categories = Category.all
+    @subregions = @region.subregions
```

Comment:
you don't have to set `@subregions` if you don't want. You can simply make those calls in the views if you'd like